reference/promise-types/edit_line.markdown

```diff
@@ -411,7 +412,37 @@ mark out the region of a file to be edited.
      }
 ```
 
-If you want to match from a starting location to the end of the file
-(even if there are other lines matching `select_start` intervening),
-then just omit the `select_end` promise and the selected region will run
-to the end of the file.
+By default end of the file will be used as end of the region if `select_end` pattern was not 
+found inside the file. If you don't want the end of the file to be implicitly matched as the end of the region, you can set [`select_end_match_eof`][bundle edit_line#select_end_match_eof] 
+value to false (true by default). 
+If the `select_end` promise will be ommited, the selected region will run to the end of the file no matter, what the value of `select_end_match_eof` is set to.
```

Comment:
Would change this sentence to "If the `select_end` attribute is omitted, the selected region will run to the end of the file no matter what the value of `select_end_match_eof` is set to."